packages/drawer/rc/__specs__/index.spec.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react'
+import { fireEvent, render } from '@testing-library/react'
+
+import { Drawer } from '../index.js'
+
+describe('Drawer', () => {
+  describe('when the arrow icons is clicked', () => {
```

Comment:
this test isn't testing the arrow buttons but the whole head el

Reply:
Copy/Pasta error from previous drawer